src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,65 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calculator {
+
+    public static String REGEX = ",|:";
+
+    public static int splitAndSum(String text) {
+        if (isBlank(text)) {
+            return 0;
+        }
+        text = getTextWithOutCustomGroup(text);
+        return sum(toInts(split(text)));
+    }
+
+    private static String getTextWithOutCustomGroup(String text) {
+        Pattern r = Pattern.compile("//(.)\n(.*)");
+        Matcher m = r.matcher(text);
+        if (m.find()) {
+            addRegex(m);
+            return m.group(2);
+        }
+        return text;
+    }
+
+    private static void addRegex(Matcher m) {
+        REGEX += "|" + m.group(1);
```

Comment:
```suggestion
        REGEX += "|" + m.group(1);
```
`"|"`협업시 해당 의미를 알기 쉽지 않을것 같습니다.
그리고 `1`에 대한 의미도 알기 쉽지 않을것 같습니다.

`1`,`"|"`에 대해 의미있는 이름을 가진 상수로 분리시켜보면 좋겠어요! 😄